src/test/ui/consts/issue-76064.rs

```diff
@@ -0,0 +1,3 @@
+struct Bug([u8; panic!(1)]); //~ ERROR panicking in constants is unstable
```

Comment:
Not sure what a good test case is. So I added the one from the issue.

Reply:
This is a good test case. This PR does not yet fix `struct Bug([u8; {let x; x}]);`, so there are no other tests to add